compiler/src/dmd/mars.d

```diff
@@ -1942,6 +1942,8 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             params.warnings = DiagnosticReporting.error;
         else if (arg == "-wi")  // https://dlang.org/dmd.html#switch-wi
             params.warnings = DiagnosticReporting.inform;
+        else if (arg == "-wi")  // https://dlang.org/dmd.html#switch-wo
```

Comment:
Looks like a copy paste in error from previous condition.

Reply:
yes it does, doesn't it!